packages/victory-core/src/index.d.ts

```diff
@@ -461,10 +461,10 @@ export interface EventPropTypeInterface<TTarget, TEventKey> {
 }
 
 export type DomainTuple = [number, number] | [Date, Date];
-export type DomainPropType =
-  | DomainTuple
+export type DomainPropObjectType =
   | { x?: DomainTuple; y: DomainTuple }
   | { x: DomainTuple; y?: DomainTuple };
+export type DomainPropType = DomainPropObjectType | DomainTuple;
```

Comment:
thank you for extracting this type out. it will make it easier to use the different domain types